library/src/main/java/pl/rspective/voucherify/android/client/VoucherifyAndroidClient.java

```diff
@@ -85,7 +88,13 @@ private VoucherifyApi createRetrofitService(Builder builder) {
         RestAdapter.Builder restBuilder =
                 new RestAdapter.Builder()
                         .setConverter(new GsonConverter(gson))
-                        .setRequestInterceptor(createInterceptor(builder.clientId, builder.clientToken, builder.origin));
+                        .setRequestInterceptor(createInterceptor(builder.clientId, builder.clientToken, builder.origin))
+                        .setErrorHandler(new ErrorHandler() {
```

Comment:
Do we have any specific errors which could be returned from server?  Maybe instead of having 'code' we could have for instance enum field with specific errors. Just thinking.....